src/app/admin/admin-sidebar/admin-sidebar.component.spec.ts

```diff
@@ -237,7 +237,18 @@ describe('AdminSidebarComponent', () => {
         expect(menuService.addSection).toHaveBeenCalledWith(comp.menuID, jasmine.objectContaining({
           parentID: 'access_control', visible: false,
         }));
+      });
+
+      it('should not show the import section', () => {
+        expect(menuService.addSection).not.toHaveBeenCalledWith(comp.menuID, jasmine.objectContaining({
+          id: 'import', visible: true,
```

Comment:
Tiny suggestion,  this probably should check that it *was called* with `visible: false` (like the other similar tests just above this one) instead of checking that it *was not called* with `visible: true`.  

Otherwise, this check here would also succeed if visible was somehow `null` or some other non-true value.

Reply:
While the `visible` property can be used to show and hide menu options, it isn't used here. In order to use it in cases where the showing or hiding of a menu option depends on a rest call (which is most cases) you'd have to: first add the option with visible false, and then wait for the response and optionally override the option with visible set to true. 
 
In practice it's less complicated to only add the option if the response indicates it should be there, and not bother with the visible property. So visible will never be false

Reply:
@artlowel : Could you are @jensvannerum add a small comment to that effect above these specs then?  It just looks odd that the specs above this all check for `visible`==`false`, while these two new ones check for `visible`!=`true`.  At a glance, that makes these tests look incorrect, but I can see now there's a reason why you had to write these tests in this manner.